00-open-a-shapefile.Rmd

```diff
@@ -78,237 +75,254 @@ Working With Shapefile Attributes in R </a>
-{ : .notice }  
+* a point shapefile representing the location of a  
+<a href="http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements" target="_blank"> flux tower</a> 
+located at the
+<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank"> NEON Harvard Forest field site</a>.
 
-The first shapefile that we will open, contains the boundary of our study area
-(or our Area Of Interest, hence the name `aoiBoundary`). 
-It is a polygon layer. We can tell it is a polygon by viewing it's `class` in
-the attributes of the object which is `SpatialPolygonsDataFrame`.
+The first shapefile that we will open contains the boundary of our study area
+(or our Area Of Interest, hence the name `aoiBoundary`). To import shapefiles we
+use the `R` function `readOGR()`.
 
 ```{r Import-Shapefile}
 #set working directory to the directory location on your computer where
-#you downloaded and unzipped the data files used in this lesson
+#you downloaded and unzipped the data files for the lesson
 #setwd("pathToDirHere")
 
-#Import a polygon shapefile 
-aoiBoundary <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
-
+#Import a polygon shapefile: readOGR("path","fileName")
+#no extension needed as readOGR only imports shapefiles
+aoiBoundary_HARV <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
 
 ```
 
-After loading a new data set we also want to view the attributes of the layer.
-This can be done simple by entering the file name. 
-
-NOTE: we can also use the `attributes()` method to view objects attributes
-{: .notice }
-
-``` {r Shapefile-attributes}
-#view attributes of the layer
-aoiBoundary
-```
+Note: The acronym, `OGR` refers to the OpenGIS Simple Features Reference 
+Implementation. 
+<a href="https://trac.osgeo.org/gdal/wiki/FAQGeneral" target="_blank"> 
+Learn more about OGR.</a> {: .notice1}  
 
-Here `R` prints a number of attributes including information about the object's `coordinate reference system (CRS)`. Note that the objects coordinates are in the projection: UTM zone 18N. This information is critical to interpreting the object extent printed on the previous line. Also, note that our object only contains a single feature. We will discuss metadata and attributes in more
-detail a bit later. Finally, we see that there is one variable named "id" with min and max values of 1.
+#Shapefile Metadata & Attributes
 
-We can look at all of the associated data attributes by printing the contents of the `data` slot with `@data`.  
+When we imported the `HarClip_UTMZ18` shapefile layer into `R` (as our
+`aoiBoundary_HARV` object), the `readOGR()` function automatically stored
+informatation about and associated with the data. We are particularly interested
+in the geospatial *metadata* that contains information on the data set and
+*attributes* that contain non-geographic information about the items in the data
```

Comment:
attributes can be geographic. Attributes which contains properties associated with each individual item stored in the vector dataset. 

Notice this text is more specific but still concise..